core/server/api/v2/utils/serializers/input/pages.js

```diff
@@ -20,6 +29,11 @@ module.exports = {
             frame.options.filter = 'page:true';
         }
 
+        if (utils.isContentAPI(frame)) {
```

Comment:
pages is only available on the content api :)

Reply:
@rishabhgrg

Reply:
🙊Updated!